examples/rsp-cra-18/src/ToDoItems.tsx

```diff
@@ -0,0 +1,41 @@
+import {CheckboxGroup, Checkbox, Flex} from '@adobe/react-spectrum'
+
+function TodoItems(props: { list : {id: number; task: string}[]; 
```

Comment:
Check out the browser console when you have some time. You'll see some warnings about missing `aria-label` or `aria-labelledy` which are important for accessibility. If you want, we can do a quick VoiceOver run to review the experience